jetcd-launcher/src/main/java/io/etcd/jetcd/launcher/EtcdContainer.java

```diff
@@ -65,7 +66,7 @@ public EtcdContainer withClusterToken(String clusterToken) {
 
     public EtcdContainer withAdditionalArgs(Collection<String> additionalArgs) {
         if (additionalArgs != null) {
-            this.additionalArgs = List.copyOf(additionalArgs);
+            this.additionalArgs = Collections.unmodifiableCollection(additionalArgs);
```

Comment:
a ocpy is needed here

Reply:
`Collections.unmodifiableCollection(new ArrayList<>(additionalArgs));` will be ok?

Reply:
yes